Jenkinsfile

```diff
@@ -444,6 +444,7 @@ stage('Default build') {
 			"""
 			dir(helper.configuration.maven.localRepositoryPath) {
 				stash name:'default-build-result', includes:"org/hibernate/search/**"
+				stash name:'default-build-cache', includes:".develocity/**"
```

Comment:
Wild thought, but you could actually get faster builds by unstashing this on all non-default environment builds... This would be useful in PR builds in particular, where we don't write to the remote build cache.

Reply:
thanks! 😃 will give it a try